packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -515,6 +523,26 @@ const helperBehavior = {
         facets.manager.burnAndRecord(accounting.toBurn, vaultSeat);
 
         // current values
+
+        // Sometimes, the AMM will sell less than all the collateral. If there
+        // was a shortfall, the investor doesn't keep the change, so we get it
+        const collateralPost = vault.getCollateralAmount();
+        if (!AmountMath.isEmpty(collateralPost)) {
+          if (!AmountMath.isEmpty(accounting.shortfall)) {
```

Comment:
Is the nesting intentional?

While this is maybe a little harder for the eyes to follow the intent is clearer and it requires less indentation:
```suggestion
        if (!AmountMath.isEmpty(collateralPost) && !AmountMath.isEmpty(accounting.shortfall)) {
```

Reply:
Not intentional. It went through a stage where all the else branches were populated. I should have recognized the opportunity to condense when I was done.